.travis.yml

```diff
@@ -1,11 +1,7 @@
+sudo: false
 language: python
-
 python:
-  - 3.6
-
-install:
-  - python -m pip install flake8 pydocstyle
```

Comment:
i see flake8 in tox. are we taking out pydocstyle checks? thats a separate thing